<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FxiWorkflowWidget</class>
 <widget class="QWidget" name="FxiWorkflowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="rotationLabelLayout"/>
     </item>
     <item row="0" column="1" rowspan="8">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="testRotationCentersGroup">
         <property name="styleSheet">
          <string notr="true">QGroupBox{padding-top:20px; margin-top:-20px}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="testRotationCentersLabel">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;Test Rotation Centers&lt;/span&gt;&lt;/h3&gt;&lt;p&gt;Perform a series of reconstructions on a single slice with a range of rotation centers, and visualize the results to determine which rotation center to use. This runs the &amp;quot;test_rotations&amp;quot; function in the script.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="startLabel">
              <property name="text">
               <string>Start:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="start">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stopLabel">
              <property name="text">
               <string>Stop:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="stop">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="stepLabel">
              <property name="text">
               <string>Steps:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="steps">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sliceLabel">
              <property name="text">
               <string>Slice:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="slice">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="testRotations">
            <property name="text">
             <string>Test Rotations</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="testRotationsSettingsGroup">
         <property name="styleSheet">
          <string notr="true">QGroupBox{padding-top:15px; margin-top:-15px}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="2">
           <widget class="QToolButton" name="colorPresetButton">
            <property name="toolTip">
             <string>Choose preset color map</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme=":/pqWidgets/Icons/pqFavorites.svg">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="currentRotation">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #F0F0F0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="tomviz::IntSliderWidget" name="imageViewSlider" native="true"/>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QGroupBox" name="previewRangesGroup">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="1">
              <widget class="tomviz::DoubleSliderWidget" name="previewMin" native="true"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="previewMaxLabel">
               <property name="text">
                <string>Max:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="tomviz::DoubleSliderWidget" name="previewMax" native="true"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="previewMinLabel">
               <property name="text">
                <string>Min:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="currentRotationLabel">
            <property name="text">
             <string>Rotation:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="testRotationCentersLabel_2">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h3 style=&quot; margin-top:4px; margin-bottom:4px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;Rotations Preview Settings&lt;/span&gt;&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="reconstructionGroup">
         <property name="styleSheet">
          <string notr="true">QGroupBox{padding-top:15px; margin-top:-15px}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="reconstructionLayout">
            <item row="2" column="0">
             <widget class="QLabel" name="sliceStartLabel">
              <property name="text">
               <string>Slice Start:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="sliceStopLabel">
              <property name="text">
               <string>Slice Stop:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>450</width>
                <height>0</height>
               </size>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;Reconstruction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="rotationCenterLabel">
              <property name="text">
               <string>Rotation Center:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sliceStart">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="sliceStop">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="tomviz::DoubleSpinBox" name="rotationCenter">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QWidget" name="additionalParametersLayoutWidget" native="true">
              <layout class="QGridLayout" name="additionalParametersLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="additionalParametersLayoutLabel">
                 <property name="minimumSize">
                  <size>
                   <width>450</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;Additional Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="tomviz::QVTKGLWidget" name="sliceView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomviz::QVTKGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKGLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>DoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>tomviz::IntSliderWidget</class>
   <extends>QWidget</extends>
   <header>IntSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::DoubleSliderWidget</class>
   <extends>QWidget</extends>
   <header>DoubleSliderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>start</tabstop>
  <tabstop>stop</tabstop>
  <tabstop>steps</tabstop>
  <tabstop>slice</tabstop>
  <tabstop>testRotations</tabstop>
  <tabstop>currentRotation</tabstop>
  <tabstop>colorPresetButton</tabstop>
  <tabstop>rotationCenter</tabstop>
  <tabstop>sliceStart</tabstop>
  <tabstop>sliceStop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
